README.md

```diff
@@ -1,169 +1,96 @@
 # kube-spawn
 
-## Introduction
-
-__kube-spawn__ is a tool for creating a multi-node Kubernetes cluster
+`kube-spawn` is a tool for creating a multi-node Kubernetes cluster
 on a single machine, created mostly for developers __of__ Kubernetes.
 
 It aims to be as similar to the solutions recommended for production
 clusters as possible.
 
-## Getting started
+## Requirements
 
-### Build proper systemd-nspawn version
+* **Host:**
+  - `systemd-nspawn` at least version 233
+  - tested: Fedora 26, ...
 
-Unfortunately, there is [one pending feature](https://github.com/systemd/systemd/pull/4395)
-of systemd-nspawn which is merged in master, but not released yet.
-You will need to build your own systemd-nspawn binary.
+* **Kubernetes** at least version 1.7.0
 
-We highly recommend using CoreOS' fork which backported that feature
-to the 231 version of systemd (which is the one that Fedora and
-the other popular distributions are using in its stable releases).
+## Quickstart
 
-In order to do that, please use the following commands:
+To quickly test `kube-spawn`, the easiest (and tested) way is to use the prepared [Vagrant setup](doc/vagrant.md).
 
-```
-git clone git@github.com:coreos/systemd.git
-cd systemd
-git checkout v231
-./autogen.sh
-./configure
-make
-```
+To setup `kube-spawn` on your machine, make sure you have a working [Go environment](https://golang.org/doc/install):
 
-You **shouldn't** do `make install` after that! Using the custom
-systemd-nspawn binary with the other components of systemd being
-in another version is totally fine.
+```
+# Get the needed CNI plugins
+$ go get -u github.com/containernetworking/plugins/plugins/main/bridge
+$ go get -u github.com/containernetworking/plugins/plugins/ipam/host-local
```

Comment:
Not sure. We tried building from vendor but that does't work if we flatten the vendor tree. And we have to do that to be able to use the docker client lib if I remember right...